src/buffer/out/textBuffer.cpp

```diff
@@ -998,6 +1002,8 @@ const TextBuffer::TextAndColor TextBuffer::GetTextForClipboard(const bool lineSe
                     selectionBkAttr.push_back(CellBkAttr);
                 }
             }
+#pragma warning(suppress : 26444)

```

Comment:
do we need to push/pop this pragma

Reply:
No.... a suppress is only active for one line. A disable is active until you enable it (or pop it back off).